packages/backend/src/core/reports/aggregateReports.test.ts

```diff
@@ -105,6 +105,7 @@ describe(aggregateReports.name, () => {
       projectId,
       tvlUsd: usdTVL * 100n,
       tvlEth: usdTVL * 1000n,
```

Comment:
Maybe we should rename those bois too? cya TVL

Reply:
Good question, we need overall aggregated TVL anyway, but the issue is every production aggregated report record is now a TVL. We can rename it now and plan the two-step migration and small refactor

Reply:
let's rename it to usdValue and ethValue

Reply:
https://github.com/l2beat/l2beat/pull/1716/commits/3e758f2fc16c7ff01bbe034331d026a95eb5e430